### ISSUE TITLE
[Suggestion] Second type of sprinkler

### ISSUE THREAD

**Issue by SlimF** [AgriCraft/AgriCraft#1188]
I really love the way sprinklers look when they work, but if you have too many of them at one place it turns in to a lag-fest. I know about the setting in the config option, but they lose their charm when they don't have any particles at all.

My suggestion is to have a second sprinkler which looks exactly the same but it doesn't show any particles when it works.

So, let's say you have 10 sprinklers which are close to each other, you can have 5 sprinkles which generate particles and 5 which do not.

**Comment by InfinityRaider** [comment]
While the particles cause overhead, having many growth ticks causing a lot of render updates creates a lot more.
A different kind of sprinkler would not change this.

**Comment by darthvader45** [comment]
Hmm, maybe finding a way to cut down on the render updates while still boosting growth?

**Comment by SlimF** [comment]
I did test it, 4 sprinklers with particles on lower my fps quite a lot. 4 sprinklers with particles off - fps is normal.

**Comment by InfinityRaider** [comment]
Then just keep the particles turned off?

**Comment by SlimF** [comment]
As I said earlier:

> 
> I know about the setting in the config option, but they lose their charm when they don't have any particles at all.
>

**Comment by AngellusMortis** [comment]
> Then just keep the particles turned off?

Is there a way to tone down the particles? I noticed this problem as well. I added `agricraft` to a Stoneblock 2 pack and I have 4 farm rooms with 4 sprinklers each. With particles on, I get sub 40 frames most of the time. With particles off, I max out my frames and the sprinklers seem to have little or no impact to my performance. 

Perhaps a `minimal` setting or something as well for the particles that just tones down how many render or something?

I am 95% certain this is specifically an `Optifine` setting as I always manually install it as well, but there is a setting under `Video Settings -> Animation Settings` for `Particles` that can be `All`, `Decreased`, `Minimal`, so may make Agricraft also respect that as well? So if you turn down particles there, also turn them down for the Sprinklers?
